apps/web/cypress/e2e/walkthroughs.cy.ts

```diff
@@ -11,17 +15,45 @@ describe("walkthrough 1", () => {
   walkthroughs.forEach((walkthrough) => {
     it(walkthrough.title, () => {
       walkthrough.steps.forEach((step) => {
-        // TODO - use test ids
         // select and submit an answer for the given question
-        cy.getByTestID(`${step.type}-${step.question}-${step.answer}`).click();
+        if (step.type === "radio") {
+          cy.getByTestID(GET_TESTID_RADIO(step.question, step.answer)).click();
+        } else if (step.type === "checkbox") {
+          step.answer.split(",").forEach((answer) => {
+            // TODO - update the application to not have duplicate test ids on checkboxes
```

Comment:
This has to do with the way aria react adds that property down the chain of elements. I'm not sure we can fix it easily from our end. What if we introduced a new helper to specifically select an input element by test id?

```javascript
Cypress.Commands.add("getInputByTestID", (id, options) => {
  return cy.get(`input[data-testid=${id}]`, options);
});
```

Reply:
I could have it deliberately get the "last" element with that ID, but still feels a little clunky. Thoughts?

Reply:
Good suggestion!